.github/workflows/build.yml.in

```diff
@@ -146,7 +143,7 @@ jobs:
         with:
           linters: gcc
           run: |
-            bash -o pipefail -c "env LEAN_ABORT_ON_PANIC=1 lake build --wfail -KCI"
+            bash -o pipefail -c "lake exe mk_all || echo "There are unaccounted files, but I am not failing yet!"; env LEAN_ABORT_ON_PANIC=1 lake build --wfail -KCI"
```

Comment:
I think the quoting needs to be adjusted: in #14779, the logs only contained the work "there".
Can you use single quotes instead (or, shudder, escape the quotes)?